openhcl/page_pool_alloc/src/lib.rs

```diff
@@ -47,27 +48,38 @@ enum State {
         pfn_bias: u64,
         #[inspect(hex)]
         size_pages: u64,
+        device_id: u64,
```

Comment:
As discussed offline, we don't want a separate ID (unless it's a temporary index into a table).